logger.js

```diff
@@ -91,6 +91,12 @@ module.exports = (externalLogger, logFolder, moduleName = "unknownmodule")=> {
       if (externalLogger) {externalLogger.log("error", detail, table, moduleName);}
       if (validUiWindow()) {uiWindow.send("errorMessage", userFriendlyMessage || detail);}
     },
+    warning(detail, userFriendlyMessage, table) {
```

Comment:
Can we get rid of userFriendlyMessage here? For the file it should just log "warning" and the warning string

Reply:
@tejohnso I've just removed userFriendlyMessage argument on the warning() function.